spot_driver/src/conversions/geometry.cpp

```diff
@@ -0,0 +1,53 @@
+
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <geometry_msgs/msg/point.hpp>
+#include <spot_driver/conversions/geometry.hpp>
+
+namespace spot_ros2::conversions {
+
+geometry_msgs::msg::Point toRosPoint(const ::bosdyn::api::Vec3& msg) {
```

Comment:
Didn't we decide to rename all these methods into `to_proto` and `to_ros`?

Reply:
Sorry, I'd fixed this but hadn't pushed up the commit.